deltachat-rpc-client/tests/conftest.py

```diff
@@ -85,11 +85,11 @@ def delete(self, uid_list: str, expunge=True):
 
     def get_all_messages(self) -> list[MailMessage]:
         assert not self._idling
-        return list(self.conn.fetch())
+        return list(self.conn.fetch(mark_seen=False))
```

Comment:
imap-tools have a weird default for the `FETCH` operation, it has `mark_seen=True` and marks all messages that we fetch with \Seen flag. This is directly in the readme: https://pypi.org/project/imap-tools/
Don't think anyone wants this even outside of tests, it is just a bad API: https://github.com/ikvk/imap_tools/issues/179